caddyhttp/index/index.go

```diff
@@ -29,6 +29,8 @@ func init() {
 func setupIndex(c *caddy.Controller) error {
 	var index []string
 
+	cnfg := httpserver.GetConfig(c)
```

Comment:
I think the usual naming is `cfg` for these

Reply:
Done